packages/node_modules/@ciscospark/internal-plugin-conversation/src/conversation.js

```diff
@@ -70,6 +70,7 @@ const Conversation = SparkPlugin.extend({
    * @param {Array<Participant>} params.participants
    * @param {Array<File>} params.files
    * @param {string} params.comment
+   * @param {string} params.markdown
```

Comment:
The sdk doesn't explicitly handle markdown; let's call this `html` to make it clear that it should already have been formatted before it hits `Conversation#create()`.